etc/bin/run.sh

```diff
@@ -30,7 +30,10 @@ KERNEL_ASSEMBLY=`(cd ${PROG_HOME}/lib; ls -1 toree-kernel-assembly-*.jar;)`
 
 # disable randomized hash for string in Python 3.3+
 export PYTHONHASHSEED=0
+# The SPARK_OPTS values during installation are stored in TOREE_SPARK_OPTS. This allows values to be specified during
+# install, but also during runtime.
+export SPARK_OPTS="${__TOREE_SPARK_OPTS__} ${SPARK_OPTS}"
 
 exec "$SPARK_HOME"/bin/spark-submit \
   ${SPARK_OPTS} \
```

Comment:
Yeah, so it looks like you are supposed to pass all the options in one flag, which we were not doing. However, the args are not in the right format and need some kind of coercing to pass them along. If I explicitly put the options in which are being passed, everything works as expected. 
Found [this](https://mail-archives.apache.org/mod_mbox/spark-dev/201404.mbox/%3CCABPQxsvpRzXyYbcjnPuvshO16V0pykdpjgs=NOdfT-FMd9Uwmw@mail.gmail.com%3E) mailing list discussion about spark submit and this problem. 
Not working yet, but it is  a step in the right direction.